### ISSUE TITLE
question: Ambiguity in the state of automation operation history

### ISSUE THREAD

**Issue by hmydgz** [apitable/apitable#1729]
## Any questions?

The execution status that needs to be added to the automation operation history is now displayed as completed when operated for a long time, and only transitions to failed after timeout

自动化运行历史需要增加的执行中状态，现在长时间操作时展示的状态是已完成，超时之后才转为失败

![image](https://github.com/apitable/apitable/assets/56059675/6437bd97-f03d-4798-b978-abe0116c1eec)

**Comment by mr-kelly** [comment]
You are absolutely right. The current state representation lacks an "In Progress" or "Running" status, which causes ambiguity for long-running automations. We will look into adding an "In Progress" state to the automation history to accurately reflect tasks that are still executing.

---

您的反馈非常准确。目前的状态展示确实缺少了“执行中 (In Progress)”的状态，这导致长时间运行的自动化任务状态存在歧义。我们会评估并在运行历史中增加“执行中”的状态，以更准确地反映任务进度。
